Examples/vmw-3ta-build.ps1

```diff
@@ -0,0 +1,388 @@
+$OvfConfiguration.networkmapping.vxw_dvs_24_universalwire_2_sid_50001_Universal_App01.value = $AppNetwork.name
+$OvfConfiguration.networkmapping.vxw_dvs_24_universalwire_3_sid_50002_Universal_Db01.value = $DbNetwork.name
+
+# VM details.
+$OvfConfiguration.common.app_ip.Value = $EdgeInternalSecondaryAddress
+$OvfConfiguration.common.Web01_IP.Value = $Web01Ip
+$OvfConfiguration.common.Web02_IP.Value = $Web02Ip
+$OvfConfiguration.common.Web_Subnet.Value = $DefaultSubnetMask
+$OvfConfiguration.common.Web_Gateway.Value = $LdrWebPrimaryAddress
+$OvfConfiguration.common.App01_IP.Value = $App01Ip
+$OvfConfiguration.common.App02_IP.Value = $App02Ip
+$OvfConfiguration.common.App_Subnet.Value = $DefaultSubnetMask
+$OvfConfiguration.common.App_Gateway.Value = $LdrAppPrimaryAddress
+$OvfConfiguration.common.DB01_IP.Value = $DB01Ip
+$OvfConfiguration.common.DB_Subnet.Value = $DefaultSubnetMask
+$OvfConfiguration.common.DB_Gateway.Value = $LdrDbPrimaryAddress
+
+# Run the deployment.
+Import-vApp -Source $BooksvAppLocation -OvfConfiguration $OvfConfiguration -Name Books -Location $ComputeCluster -VMHost $Vmhost -Datastore $ComputeDatastore | out-null
+write-host -foregroundcolor "Green" "Starting $vAppName vApp components"
+Start-vApp $vAppName | out-null
+
+
+#####################################
+# Microseg config
+
+write-host -foregroundcolor Green "Getting Services"
+
+# Assume these services exist which they do in a default NSX deployment.
+$httpservice = New-NsxService -name "tcp-80" -protocol tcp -port "80"
+$mysqlservice = New-NsxService -name "tcp-3306" -protocol tcp -port "3306"
+
+#Create Security Tags
+
+$WebSt = New-NsxSecurityTag -name $WebStName
+$AppSt = New-NsxSecurityTag -name $AppStName
+$DbSt = New-NsxSecurityTag -name $DbStName
+
+
+# Create IP Sets
+
+write-host -foregroundcolor "Green" "Creating Source IP Groups"
+$AppVIP_IpSet = New-NsxIPSet -Name AppVIP_IpSet -IPAddresses $EdgeInternalSecondaryAddress
+$InternalESG_IpSet = New-NsxIPSet -name InternalESG_IpSet -IPAddresses $EdgeInternalPrimaryAddress
+
+write-host -foregroundcolor "Green" "Creating Security Groups"
+
+#Create SecurityGroups and with static includes
+$WebSg = New-NsxSecurityGroup -name $WebSgName -description $WebSgDescription -includemember $WebSt
+$AppSg = New-NsxSecurityGroup -name $AppSgName -description $AppSgDescription -includemember $AppSt
+$DbSg = New-NsxSecurityGroup -name $DbSgName -description $DbSgDescription -includemember $DbSt
+$BooksSg = New-NsxSecurityGroup -name $BooksSgName -description $BooksSgName -includemember $WebSg, $AppSg, $DbSg
+
+# Apply Security Tag to VM's for Security Group membership
+
+$WebVMs = Get-Vm | ? {$_.name -match ("Web0")}
+$AppVMs = Get-Vm | ? {$_.name -match ("App0")}
+$DbVMs = Get-Vm | ? {$_.name -match ("Db0")}
+
+
+Get-NsxSecurityTag $WebStName | New-NsxSecurityTagAssignment -ApplyToVm -VirtualMachine $WebVMs | Out-Null
```

Comment:
Likewise earlier comment - use the already defined variables rather than regetting...